glide.yaml

```diff
@@ -1,6 +1,7 @@
 package: github.com/eris-ltd/eris-db
 import:
 - package: github.com/tendermint/tendermint
+  version: 55ef4b225fb0aa5637a96d36a8c0d030a59dc21d
```

Comment:
it will be interesting to see how this plays out with the independent tendermint/go-* repositories, but :+1: